packages/mui-base/src/PopperUnstyled/PopperUnstyled.types.ts

```diff
@@ -6,7 +6,7 @@ import { SlotComponentProps } from '../utils';
 
 export type PopperPlacementType = Options['placement'];
 
-interface PopperUnstyledComponentsPropsOverrides {}
+interface PopperUnstyledRootSlotPropsOverrides {}
```

Comment:
Not sure if it's related to this PR, but shouldn't this interface be exported?

We've encountered an issue when upgrading MUI Core packages in https://github.com/mui/mui-x/pull/7905:
https://ci.codesandbox.io/status/mui/mui-x/pr/7905/builds/349844

```
cmd: 'yarn tsc -b /tmp/fad0ee40/packages/grid/x-data-grid-generator/tsconfig.build.json',
  stdout: '$ /tmp/fad0ee40/node_modules/.bin/tsc -b /tmp/fad0ee40/packages/grid/x-data-grid-generator/tsconfig.build.json\n' +
    `../x-data-grid/src/components/panel/GridPanel.tsx(38,7): error TS4023: Exported variable 'GridPanelRoot' has or is using name 'PopperUnstyledRootSlotPropsOverrides' from external module "/tmp/fad0ee40/node_modules/@mui/base/PopperUnstyled/PopperUnstyled.types" but cannot be named.\n` +
```

cc @michaldudak

Reply:
Yes, absolutely. Thanks for letting me know. I'll review other cases and fix it right away.